src/pages/workspace/companyCards/WorkspaceCompanyCardAccountSelectCardPage.tsx

```diff
@@ -33,7 +33,7 @@ function WorkspaceCompanyCardAccountSelectCardPage({route}: WorkspaceCompanyCard
     const workspaceAccountID = PolicyUtils.getWorkspaceAccountID(policyID);
     const [searchText, setSearchText] = useState('');
 
-    const [allBankCards] = useOnyx(`${ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST}${workspaceAccountID}_${bank}`);
+    const [allBankCards] = useOnyx(`${ONYXKEYS.CARD_LIST}`);
```

Comment:
@narefyev91 Could you explain this change? I think using ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST is correct on this page. please correct me If I am wrong

Reply:
@DylanDylann on src/pages/workspace/companyCards/WorkspaceCompanyCardDetailsPage.tsx - we show information which is coming from here:
<img width="949" alt="Screenshot 2024-11-08 at 10 54 03" src="https://github.com/user-attachments/assets/d1860ece-3b69-4290-a51a-39eeb0350ccb">

That's why any changes using ONYXKEYS.COLLECTION.WORKSPACE_CARDS_LIST was not shown on WorkspaceCompanyCardDetailsPage - because we are not using that key for showing card details